extensions/qotp/plugins.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "QOTP",
+  "description": "生成 OTP 六位动态码",
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.